docs/redoc/master/openapi.json

```
@@ -230,6 +230,84 @@
         }
       }
     },
+    "/healthz": {
+      "get": {
+        "summary": "Kubernets healthz endpoint",
```

```suggestion
        "summary": "Kubernetes healthz endpoint",
```

The typo is present 3 times :grin: